scripts/plotting/postfitPlots.py

```diff
@@ -47,6 +47,9 @@
 else:
     raise IOError("Unknown format of input file")
 
+# order of the processes in the plots
+procs_sort = ["Wmunu", "Fake", "Zmumu", "Wtaunu", "Top", "DYlowMass", "Other", "Ztautau", "Diboson"][::-1]
```

Comment:
This can be read from the groups can't it?

Reply:
The script doesn't have a datagroups object and because of that can't access the ordering that is defined when creating the group